src/Factory/RabbitMqWorkerControllerFactory.php

```diff
@@ -29,7 +29,7 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
      */
     public function createService(ServiceLocatorInterface $serviceLocator, $canonicalName = null, $requestedName = null)
     {
-        /** @var \Zend\Mvc\Controller\ControllerManager $serviceLocator*/
+        /** @var \Laminas\Mvc\Controller\ControllerManager $serviceLocator*/
 
         return $this($serviceLocator->getServiceLocator(), RabbitMqWorkerController::class);
     }
```

Comment:
This method is not used as it does not belong to Laminas\ServiceManager\Factory\FactoryInterface interface.
Remove it including corresponding class imports.